docs/application/toc_all.md

```diff
@@ -1,62 +1,67 @@
 # Get Started
 
 ## [Overview](/application/index.md)
-## [Wearable](/application/get-started/wearable.md)
-## [TV](/application/get-started/tv.md)
-## [Mobile](/application/get-started/mobile.md)
-## [IoT](/application/get-started/iot.md)
-## [Tizen Applications](/application/get-started/applications.md)
 
+## Tizen Profiles
+
+### [Overview](/application/profiles/index.md)
+### [Wearable](/application/profiles/wearable.md)
+### [Mobile](/application/profiles/mobile.md)
+### [IoT](/application/profiles/iot.md)
+### [TV](/application/profiles/tv.md)
 
 # .NET Application
 
 
 ## [Overview](/application/dotnet/index.md)
-
+## [Learn about .NET](/application/dotnet/learn-about-dotnet.md)
 
 ## Get Started
 
-### [Overview](/application/dotnet/get-started/overview.md)
+### [Overview](/application/dotnet/get-started/index.md)
 ### [Create Wearable .NET Application](/application/dotnet/get-started/wearable/first-app.md)
 ### [Create Wearable XAML Application](/application/dotnet/get-started/wearable/first-xaml-app.md)
 ### [Create TV .NET Application](/application/dotnet/get-started/tv/first-app.md)
 ### [Create Mobile .NET Application](/application/dotnet/get-started/mobile/first-app.md)
-### [Create NUI .NET Application](/application/dotnet/get-started/nui/quickstart.md)
+### [Create NUI .NET Application](/application/dotnet/get-started/nui/first-app.md)
 ### [Create IoT .NET Application](/application/dotnet/get-started/iot/first-app.md)
+### [API Privileges](/application/dotnet/get-started/api-privileges.md)
+### [Hybrid Application](/application/dotnet/get-started/hybrid-application.md)
+### Certificates
+#### [Overview](/application/dotnet/get-started/certificates/index.md)
+#### [Installing the Extension](/application/dotnet/get-started/certificates/installing-the-extension.md)
+#### [Creating Certificates](/application/dotnet/get-started/certificates/creating-certificates.md)
+#### [Signing Application with Certificate](/application/dotnet/get-started/certificates/signing-application-with-certificate.md)
+#### [Managing Certificate Profile](/application/dotnet/get-started/certificates/managing-certificate-profile.md)
+### [Wearable Connection](/application/dotnet/get-started/wearable-connection.md)
```

Comment:
Though this link seem to be working fine in staging, request to use the accurate link:
### [Wearable Connection](/application/dotnet/get-started/wearable-connection/index.md)

Reply:
Link has been updated.

Reply:
@DariuszPaziewski, this link is still not updated